src/compiler/crystal/config.cr

```diff
@@ -5,7 +5,7 @@ module Crystal
     end
 
     def self.version
-      {{ read_file("#{__DIR__}/../../../VERSION").chomp }}
+      {{ (read_file?("#{__DIR__}/../../../VERSION") || "").chomp }}
```

Comment:
That's incorrect. If there's no `VERSION` file it should return `nil`.

Reply:
That's how CRYSTAL_PATH has done